packages/cli/src/lib/bundler/config.ts

```diff
@@ -232,12 +233,52 @@ export async function createConfig(
     require.resolve('react-refresh'),
   ];
 
+  const mode = isDev ? 'development' : 'production';
+  const optimization = optimizationConfig(options);
+
+  if (
+    !!process.env.EXPERIMENTAL_MODULE_FEDERATION &&
+    !!process.env.FORCE_REACT_DEVELOPMENT
```

Comment:
```suggestion
    process.env.EXPERIMENTAL_MODULE_FEDERATION &&
    process.env.FORCE_REACT_DEVELOPMENT
```

!!!

Reply:
if you really do want to make this very explicit, it'd opt for `Boolean(...)` instead

Reply:
Sure 😁 6c94522688dc